### ISSUE TITLE
Questions about test results

### ISSUE THREAD

**Issue by Halleyawoo** [cchen-cc/SIFA#26]
Hello, author, when I tested the CT data set, the results obtained were quite different from the results in the paper. I didn't change any settings, and I don't know what went wrong.
![image](https://user-images.githubusercontent.com/46990176/99880361-ddc96780-2c4d-11eb-830d-92462eab5e7a.png)

**Comment by Halleyawoo** [comment]
Dice:
AA :0.0(0.0)
LAC:0.8(0.9)
LVC:6.8(3.3)
Myo:4.0(3.1)
Mean:2.9
ASSD:
AA :50.0(3.5)
LAC:42.2(13.0)
LVC:31.7(2.3)
Myo:29.8(4.1)
Mean:38.4

**Comment by Halleyawoo** [comment]
I'm very sorry for being careless. Solved.

**Comment by davidemilesi** [comment]
> I'm very sorry for being careless. Solved.

Hello, first I want to thanks to authors for sharing this work.

I obtained similar metrics with preprocessed data and pretrained weights. The input and label data has correct appearance, but the predicted labels seems to have a random distribution in the volume. Could I ask how you solved it?

Also, in this other [issue](https://github.com/cchen-cc/SIFA/issues/27#issuecomment-944354051) you said that changing max_step and bacth_size helped you successfully train the network. Is there a particular reason for this choice? Did you happen to understand why the original settings don't work?

Thank you in advance @Halleyawoo
